### PR TITLE
fix: Add metadata to property to product category data model

### PR DESCRIPTION
**Pull request by Alexnortung**

### CODE REVIEW

**Review by sradevski — APPROVED** [review]
LGTM 👍

File: packages/modules/product/src/models/product-category.ts [hunk 1]
```diff
@@ -112,6 +112,9 @@ class ProductCategory {
   @Property({ columnType: "timestamptz", nullable: true })
   deleted_at?: Date
 
+  @Property({ columnType: "jsonb", nullable: true })
+  metadata?: Record<string, unknown> | null
```
Comment (riqwan) [inline packages/modules/product/src/models/product-category.ts:116]:
```suggestion
  metadata: Record<string, unknown> | null = null
```

**Comment by olivermrbl** [comment]
Thanks for contribution @Alexnortung

File: packages/modules/product/src/models/product-category.ts (hunk 1 above)
Comment (riqwan) [reply packages/modules/product/src/models/product-category.ts:116]: You also need to add this to the types:

```
metadata?: Record<string, unknown> | null
```
 file: `https://github.com/medusajs/medusa/blob/develop/packages/core/types/src/http/product-category/common.ts#L5`

**Review by riqwan — APPROVED** [review]
one comment, but lgtm!

**Comment by riqwan** [comment]
Sorry, bit late to the party 😄

File: packages/modules/product/src/models/product-category.ts (hunk 1 above)
Comment (sradevski) [reply packages/modules/product/src/models/product-category.ts:116]: Yeah we mentioned it in Slack but Oli merged already 😄

Comment (olivermrbl) [reply packages/modules/product/src/models/product-category.ts:116]: Sorry must have missed that.

**Pull request merged**